blocks/inspector-controls/range-control/style.scss

```diff
@@ -3,12 +3,75 @@
 	justify-content: center;
 }
 
+// creating mixin because we can't do multiline variables, and we can't comma-group the selectors for styling the range slider
+@mixin range-thumb() {
+	height: 18px;
+	width: 18px;
+	border-radius: 50%;
+	cursor: pointer;
+	background: $dark-gray-500;
+	border: 4px solid transparent;
+	background-clip: padding-box;
+}
+
+@mixin range-track() {
+	height: 3px;
+	cursor: pointer;
+	background: $light-gray-500;
+	border-radius: 1.5px;
```

Comment:
I did that initially, but because the track isn't a square that means the whole track becomes an oval.